web/usuaris/src/components/organisms/SideMenu.tsx

```diff
@@ -80,9 +80,6 @@ const CategoryStyled = styled.span`
 const ImgStyled = styled(Icon)`
   height: 30px;
   margin-right: ${dimensions.spacing.xxxs};
-
-  &[data-testid] {
-    content: attr(data-testid);
```

Comment:
These lines there is no need to touch them. You could undo the changes on this file to not appear in this pr